compiler/passes/convert-uast.cpp

```diff
@@ -1591,6 +1590,40 @@ struct Converter {
     return ret;
   }
 
+  Expr* convertLifetimeClause(const uast::Expression* node) {
+    assert(node->isOpCall() || node->isReturn());
+    if (auto opCall = node->toOpCall()) {
+      assert(opCall->numActuals()==2);
+      auto lhsIdent = opCall->actual(0)->toIdentifier();
+      auto rhsIdent = opCall->actual(1)->toIdentifier();
+      assert(lhsIdent && rhsIdent);
+      assert(opCall->op() == "=" || opCall->op() == "<" || opCall->op() == "<="
```

Comment:
I know this is just an assert but I'd recommend `opCall->op() == USTR("=")` etc so that we can get used to using USTR.